sdk/metric/meter_test.go

```diff
@@ -2427,3 +2427,19 @@ func TestDuplicateInstrumentCreation(t *testing.T) {
 		})
 	}
 }
+
+func TestMeterProviderDelegation(t *testing.T) {
+	meter := otel.Meter("go.opentelemetry.io/otel/metric/internal/global/meter_test")
+	otel.SetErrorHandler(otel.ErrorHandlerFunc(func(err error) { require.NoError(t, err) }))
+	for i := 0; i < 5; i++ {
+		m, err := meter.Int64ObservableUpDownCounter("observable.int64.up.down.counter")
```

Comment:
This is only testing an `Int64ObservableUpDownCounter`. Can we test all the instruments that are being fixed in the PR?

Reply:
Can you wrap this in [`assert.NotPanics`](https://pkg.go.dev/github.com/stretchr/testify/assert#NotPanics) to indicate what this is testing?